sycl/doc/extensions/experimental/sycl_ext_oneapi_bfloat16_math_functions.asciidoc

```diff
@@ -104,6 +104,23 @@ The following functions are only available when `T` is `bfloat16` or
 `sycl::marray<bfloat16, {N}>`, where `{N}` means any positive value of
 `size_t` type.
 
+==== isnan
+
+```c++
+namespace sycl::ext::oneapi::experimental {
+
+template <typename T>
+bool isnan(T x);
```

Comment:
```suggestion
bool isnan(bfloat16 x);
```

The paragraph above says that `T` must either be `bfloat16` or `sycl::marray<bfloat16, {N}>`.  For `isnan`, there are different overloads for these two cases, so I don't think you want to use a template with `T` here.